ui/deferredtreeview.cpp

```diff
@@ -56,6 +56,17 @@ void setSectionResizeMode(QHeaderView *header, int logicalIndex, QHeaderView::Re
     header->setResizeMode(logicalIndex, mode);
 #endif
 }
+
+QStyle::StateFlag checkStateStyleFlag(Qt::CheckState state) {
+    switch (state) {
+    case Qt::Unchecked:
+        return QStyle::State_Off;
+    case Qt::PartiallyChecked:
+        return QStyle::State_NoChange;
+    case Qt::Checked:
+        return QStyle::State_On;
+    }
+}
```

Comment:
GCC leaves me with a 
`/home/christoph/KDAB/QT/GammaRay/ui/deferredtreeview.cpp:69:1: error: control reaches end of non-void function [-Werror=return-type]` here. Maybe add a default?